site/content/usage/20-schema.md

```diff
@@ -216,6 +227,21 @@ ListMeta:
     selfLink:
       type: string
   type: object
+ManagedFieldsEntry:
```

Comment:
Where did this changes come from?

Reply:
This comes from the updated `metav1.ObjectMeta` struct that has changed between Kubernetes 1.12.x and 1.14.x.

The whole chain looks like:

- We use `ObjectMeta` in `ClusterIAMServiceAccount`:
    ```go
    type ClusterIAMServiceAccount struct {
            metav1.ObjectMeta `json:"metadata,omitempty"`
            // +optional
            AttachPolicyARNs []string `json:"attachPolicyARNs,omitempty"`
            // +optional
            AttachPolicy InlineDocument `json:"attachPolicy,omitempty"`
            // +optional
            Status *ClusterIAMServiceAccountStatus `json:"status,omitempty"`
    }
    ```
- `ObjectMeta` has a new field `managedFields`: https://github.com/kubernetes/apimachinery/blob/a9f1d8a9c10182d101acf19b5145c3d4e9299adb/pkg/apis/meta/v1/types.go#L267). This is reflected in the schema of `ObjectMeta`

- This new field is of type `ManagedFieldsEntry`, which needs the `Fields` type. Both of those are now in the schema as definitions.

Reply:
Aha, ok. Thanks for the detailed explanation :) I guess we shouldn't be reusing third-party objects in our API... :/ 

Please rebase and fix the conflicts and I'll approve :+1: